src/networking/controllers/user-controller.ts

```diff
@@ -12,6 +12,16 @@ class UserController {
     const response = await ApiService.get<RawUsers>(API_ROUTES.USERS, { page }, config);
     return UserSerializer.deSerialize(response.data);
   }
+
+  static async getUser(id: string) : Promise<FullDummyUser> {
+    const config = {
+      headers: {
+        'app-id': '635aa88d7338f9f2167b2ab4',
```

Comment:
this would be a good value to keep in a .env variable

Reply:
you should use process.env.APP_ID